.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "contrib/openmc"]
 	path = contrib/openmc
-	url = https://github.com/openmc-dev/openmc.git
+	url = https://github.com/pshriwise/openmc.git
```

Comment:
I'll open a PR soon with a `UserObject` that should let us move to the latest OpenMC release and stay off my fork. Do we want to push that in first and leave the submodule alone?

Reply:
I'd prefer to keep on the mainline openmc if possible

Reply:
I had no intentions of switching to Patrick's fork; this was just to test things!

Reply:
I think you'd said that before and I forgot -- my bad!